client/src/pages/Home.css

```diff
@@ -9,3 +9,8 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.main-footer {
+	background-color: dimgray;
```

Comment:
I suggest we create each component with its accompanying css, this is workable now but will become very confusing if we have a lot of custom styles in the future, you can create a Footer.css file and use that if possible